### DISCUSSION TITLE
Audiogridder does not work at all

### DISCUSSION THREAD

**Discussion by Thebenes** [apohl79/audiogridder#1402] [General]
Hello. Audiogridder does not work at all on my system. There is a latency of several tens of seconds and I only hear very short noises. My DAW is also very slow, even when the plugin is very light on CPU
However, I have correctly installed Audiogridder on both my computers: the “server” on my laptop, which runs under Windows 11, and the "plugin" on my desktop, which runs under Windows 10. My DAW is Reaper. I've connected my two computers via ethernet to a Gigabit router, which is connected to my Internet box. I have the impression that my configuration is perfectly compliant, but I don't understand why it's not working.

**Comment by bluetardis** [comment]
Can you ping the ip address of the other computer?
Are there any windows firewall or other stuff going on?

**Comment by Thebenes** [comment]
Thanks for your reply. You were right, I deactivated the firewall on the local network of my two computers and the ping responded. 
However, my problem hasn't been solved because Audiogridder still doesn't work properly. The plug-in interface window keeps closing and opening. There's still no sound, apart from very short glitches, and the CPU is really struggling.

**Comment by bluetardis** [comment]
Something basic isn't right.   I tend to suspect Windows networking somewhere... but its hard to diagnose remotely.

Things you can try:
-Get what's called a "cross-over" cable.  You can use this to plug your computer directly into your laptop (no router).
-You will likely need to assign an IP address for each device eg 192.168.1.10 for the server and 192.168.1.11 for the computer.  If it asks for a net mask then 255.255.0.0 and don't worry about a gateway.

A fix might be to check the buffer size - increase it if you have nothing but generally it will be fine with whatever its defaulted to.

Turn off any other apps that are running (especially screensavers).

On the server turn on the statistics and see what's happening.  Is the CPU spiking on server or client?  Is the network traffic looking reasonable according to the server statistics.?

Something in your setup isn't stable and by going through these steps it might clue you in on what it is.

**Comment by Thebenes** [comment]
On my main computer I have an ethernet socket for my internet connection, and I have an Intel AMT socket. Can I use this socket to connect my second computer with the crossover cable?

**Reply by bluetardis** [reply]
You can do anything but... initially you are trying to keep things simple so you can troubleshoot and work out what wasn't working.  Once you have that sorted then plugin however you want

**Reply by Thebenes** [reply]
Yes of course, I misspoke. What I meant was that I only have two ethernet sockets: one for the internet, the other is the AMT. If I plugged my second computer into the first ethernet socket, I wouldn't have any internet. That's why using a switch seemed like a practical solution to be able to connect my two computers and still have Internet. But I'm going to test your solution to get a clearer idea, as soon as I've found a crossover cable. 

Translated with www.DeepL.com/Translator (free version)

**Comment by Thebenes** [comment]
I did a few tests with the cables I had at the time. On my main computer, I tried each of my two ethernet sockets. When I'm connected to the first port, the Audiogridder server window keeps opening and closing. When I'm connected to the AMT port, the server window remains stable, but the sound still doesn't work.
On the other hand, I tried your advice: I changed the IP addresses of my two computers, but it didn't do any good, the computers weren't communicating and even the pings weren't getting through. So I reset the automatic values.
For my two computers, I saw in the task manager that the network usage was around 100%. The processor and memory were doing quite well, with fairly low values. 
Of course I've disabled the Windows firewall on both my computers, and I don't have any screensaver. On the other hand, I couldn't find a crossover cable, I did it all with a straight cable. I'll test again as soon as I find one.

**Reply by bluetardis** [reply]
The Cross Over cable is just a different wiring that will enable you to plug two computers together without a network switch etc.
Some computers can detect that you have another computer plugged in and will automatically sort the wiring but most will require a crossover cable.  It will also let you connect at the best speed your network cards will support vs the network switch and most of them are 100.

As far as IP addressing etc. I would go with a seperate network setup for testing.  You can always make it "automatic" which will get whatever your router/modem hands on when you have finished testing.
A safe bet is  something like:
computer01 = 192.168.2.10
computer02 = 192.168.2.11
mask = 255.255.0.0

That should let them ping each other etc.

No idea why network usage was 100%.

>

**Comment by Thebenes** [comment]
Hi, I tried today with a crossover cable. At first it worked almost fine, I managed to turn a reverb remotely. There were still a few glitches but it was very encouraging. Then I tried changing the IP addresses as you advised, thinking that would improve things. The pings went through perfectly in both directions. But audiogridder wasn't working properly at all. And then, no matter what I did (reset the IP values to default or to manual, restart my computers, etc), it still didn't work.
I'm starting to think about giving up...

**Comment by bluetardis** [comment]
Remember - if you changed to the IP addresses I gave you, you will need to enter a new server.
So try one more time.
-use the crossover cable
-set the ip addresses 
-make sure firewall etc is off
-confirm that you can ping.
-reboot!

Then restart the audiogridder server and client
you will likely need to add the server again from your client/daw when you start the plugin < <------ I think this is what's missing but its good to check it all

**Comment by Thebenes** [comment]
I tried it all in exactly this order. I checked that the IP addresses were the right ones, I checked that the ping passed in both directions (3ms), I deactivated all my firewalls on both computers, then I rebooted them. On the second one I cut the wifi connection (because I saw that it could cause problems) and I launched the audiogridder server. On the 1st computer I launched Reaper, then I opened the plugin agridder, then I checked the server IP 192.168.1.11 was selected, then I launched the little reverb installed on my second computer. Result: it doesn't work at all. No sound, Reaper runs in slow motion.

**Comment by Thebenes** [comment]
Even adding a new server 192.168.1.11 in the plugin doesn't work. Even if I don't run any vst remotely. No sound. RT CPU consumption: 1922.2%, CPU: 2%, network used at 98% according to the task manager, and at 100% on my second computer, even though I'm not running any VST!

**Comment by Thebenes** [comment]
I tried again with the IP values assigned automatically. After rebooting both machines, the ping was fine and the agridder plugin immediately found the server with the new IP address. But there was still no sound and my DAW was completely blocked.

**Comment by bluetardis** [comment]
Hey mate, I will try and get a screenshot for you when I am back in but 3ms seems long ping time.
I don't know what else to say/try at this point.  What you are seeing isn't normal behaviour but I don't know what else to try and suggest.  

I will update with a ping screen capture in a couple of hours.

**Comment by Thebenes** [comment]
Ihave to say that my second computer is a laptop that doesn't have an ethernet port. So I use an ethernet -usb adapter, then a second adapter for usb c. This may complicate the connection, but I don't think so, because I tested it for the Internet and it worked perfectly well. Thanks again for your help.

Traduit avec DeepL https://www.deepl.com/app/?utm_source=android&utm_medium=app&utm_campaign=share-translation

Le 31 octobre 2024 04:04:45 GMT+01:00, Peter ***@***.***> a écrit :

**Reply by bluetardis** [reply]
Here are my ping times from one computer to the other one.  
This is using CAT6 cables and a cheap UniFi Network Switch (though I think anything decent will be ok).

**Note that the average ping is 0.8ms when under load.**  
A 5ms ping and full network traffic means you are getting errors and retries that are at best slowing down the connection and at worst making it unusable.

```
Mac-Studio ~ % ping 192.168.2.114
PING 192.168.2.114 (192.168.2.114): 56 data bytes
64 bytes from 192.168.2.114: icmp_seq=0 ttl=64 time=0.671 ms
64 bytes from 192.168.2.114: icmp_seq=1 ttl=64 time=0.760 ms
64 bytes from 192.168.2.114: icmp_seq=2 ttl=64 time=0.800 ms
64 bytes from 192.168.2.114: icmp_seq=3 ttl=64 time=0.841 ms
64 bytes from 192.168.2.114: icmp_seq=4 ttl=64 time=0.818 ms
64 bytes from 192.168.2.114: icmp_seq=5 ttl=64 time=0.862 ms
64 bytes from 192.168.2.114: icmp_seq=6 ttl=64 time=0.797 ms
64 bytes from 192.168.2.114: icmp_seq=7 ttl=64 time=0.907 ms
^C
--- 192.168.2.114 ping statistics ---
8 packets transmitted, 8 packets received, 0.0% packet loss
round-trip min/avg/max/stddev = 0.671/0.807/0.907/0.066 ms
```
